libs/shared/components/src/auth/UserMenu/UserMenu.tsx

```diff
@@ -1,10 +1,15 @@
 import React, { useState } from 'react'
-import { Box } from '@island.is/island-ui/core'
+import { Box, Button } from '@island.is/island-ui/core'
 import { useAuth } from '@island.is/auth/react'
-import { UserDropdown } from './UserDropdown'
 import { UserButton } from './UserButton'
+import { UserDropdown } from './UserDropdown'
+import { UserLanguageSwitcher } from './UserLanguageSwitcher'
 
-export const UserMenu = () => {
+export const UserMenu = ({
+  showLanguageButton = false,
+}: {
+  showLanguageButton?: boolean
+}) => {
```

Comment:
This PR is missing a language picker in the application system form. Space is more tight there, which is why I liked having the language picker (consistently) in the user menu.

Thinking was that the user is unlikely to change this part of their user profile many times so it doesn't need to take space on every screen when you're logged in.

![image](https://user-images.githubusercontent.com/115094/146916094-3b01b95a-4326-4d75-8a72-56cd19b392bf.png)

But maybe there is space for the to fall into one more line, vertically centered.

Reply:
I see - The language part was taken from UserMenu and the landing was to make it more like the main web, spoke to @hrefnalind about this. I totally forgot about this part. I'll view this and put it back in under condition?

Reply:
Yeah, I think it is more important for it to be consistent with other places where a user is logged in than the main web. But if we can have it globally consistent, then great I guess? :)

Reply:
Done ✅  - Fixed with lang button on desktop and lang select box on mobile